### PR TITLE
Adding images in Test report

### PR DESCRIPTION
**Pull request by shashwattrivedi**

### CODE REVIEW

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 [hunk 1]
```diff
@@ -51,9 +51,22 @@ $AADCommonIssue =
...
+</br>
+Run 
+</br>
+Resolve-DnsName login.microsoftonline.com
+</br> (or equivalent FQDN for the Azure environment in powershell) on the machine and verify if you are able to resolve the uri. </li>
+<li> Check Network Security Group (NSG) attached to virtual machine's subnet, NSG should allow outbound connection to AzureActiveDirectory <a href = 'https://docs.microsoft.com/azure/virtual-network/service-tags-overview'>Service Tags</a>
+</br>
+Network Security Groups for a virtual machine can be checked at Azure portal Virtual machine blade -> Networking Settings -> Network Interface -> Outbound port rules. <IMG SRC='media/VMNetworkblade.png' ALT='Virtual machine NSG outbound port rules' width=1024 height=480/>
+</br>
+If the rule is missing, Click on 'Add outbound port rule' and fill in the source as Virtual network, destination as Service tag and Destination Service tag as AzureActiveDirectory. Set Action as Allow and click Add.
```
Comment (kajai987) [inline ConnectivityTestScriptsForWindows/Publish-Report.ps1:64]: **c**lick on 'Add..'

Comment (kajai987) [inline ConnectivityTestScriptsForWindows/Publish-Report.ps1:64]: 'Service tag' and 'AzureActiveDirectory'? (within quotes)

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 [hunk 2]
```diff
@@ -51,9 +51,22 @@ $AADCommonIssue =
...
+Specific ports for this connection can be found under sections 56 and 59 in <a href='https://docs.microsoft.com/microsoft-365/enterprise/urls-and-ip-address-ranges?view=o365-worldwide&preserve-view=true#microsoft-365-common-and-office-online'> Microsoft 365 Common and Office Online .</a>
```
Comment (kajai987) [inline ConnectivityTestScriptsForWindows/Publish-Report.ps1:65]: remove space at the end

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 [hunk 3]
```diff
@@ -51,9 +51,22 @@ $AADCommonIssue =
 <div style='background-color:#FF033E;color:white;padding:20px;'>
 <h2>Azure Active Directory (AAD) Common Issue</h2>
 <ul>
-<li> Check resolution for AAD FQDN login.microsoftonline.com (Azure AD Global service), login.microsoftonline.us (Azure AD for US Government) , login.partner.microsoftonline.cn (Azure AD China)</li>
-<li> Check Network Security Group (NSG) attached to subnet, NSG should allow outbound connection to Service Tag AAD <a href = 'https://docs.microsoft.com/azure/virtual-network/service-tags-overview'>Service Tags</a></li>
-Check firewall settings for outbound connection to URLs mentioned under sections 56 and 59 in <a href='https://docs.microsoft.com/microsoft-365/enterprise/urls-and-ip-address-ranges?view=o365-worldwide&preserve-view=true#microsoft-365-common-and-office-online'> Microsoft 365 Common and Office Online .</a> </li>
+<li> Check DNS resolution for Azure active directory FQDN login.microsoftonline.com (Azure AD Global service), login.microsoftonline.us (Azure AD for US Government) , login.partner.microsoftonline.cn (Azure AD China).
```
Comment (kajai987) [inline ConnectivityTestScriptsForWindows/Publish-Report.ps1:54]: Check DNS resolution for Azure active directory FQDN. Use
  - login.microsoftonline.com (Azure AD Global service)
  - login.microsoftonline.us (Azure AD for US Government)
  - login.partner.microsoftonline.cn (Azure AD for China)

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 [hunk 4]
```diff
@@ -51,9 +51,22 @@ $AADCommonIssue =
...
 <ul>
-<li> Check resolution for AAD FQDN login.microsoftonline.com (Azure AD Global service), login.microsoftonline.us (Azure AD for US Government) , login.partner.microsoftonline.cn (Azure AD China)</li>
-<li> Check Network Security Group (NSG) attached to subnet, NSG should allow outbound connection to Service Tag AAD <a href = 'https://docs.microsoft.com/azure/virtual-network/service-tags-overview'>Service Tags</a></li>
-Check firewall settings for outbound connection to URLs mentioned under sections 56 and 59 in <a href='https://docs.microsoft.com/microsoft-365/enterprise/urls-and-ip-address-ranges?view=o365-worldwide&preserve-view=true#microsoft-365-common-and-office-online'> Microsoft 365 Common and Office Online .</a> </li>
+<li> Check DNS resolution for Azure active directory FQDN login.microsoftonline.com (Azure AD Global service), login.microsoftonline.us (Azure AD for US Government) , login.partner.microsoftonline.cn (Azure AD China). 
+</br>
+Run 
+</br>
+Resolve-DnsName login.microsoftonline.com
+</br> (or equivalent FQDN for the Azure environment in powershell) on the machine and verify if you are able to resolve the uri. </li>
```
Comment (kajai987) [inline ConnectivityTestScriptsForWindows/Publish-Report.ps1:59]: verify if the uri resolution goes through.

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 (hunk 1 above)
Comment (kajai987) [inline ConnectivityTestScriptsForWindows/Publish-Report.ps1:64]: use consistent casing for field names, check what is commonly used and appropriate

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 [hunk 5]
```diff
@@ -65,8 +78,16 @@ $AzureBackupCommonIssue =
 <ul>
 <li> <b>Azure Workload backup doesn't support authenticated Proxy servers. <a href ='https://docs.microsoft.com/azure/backup/backup-sql-server-database-azure-vms#use-an-http-proxy-server-to-route-traffic'> Use an HTTP proxy server to route traffic</a> </b></li>
```
Comment (kajai987) [inline ConnectivityTestScriptsForWindows/Publish-Report.ps1:79]: Azure Workload Backup everywhere

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 [hunk 6]
```diff
@@ -65,8 +78,16 @@ $AzureBackupCommonIssue =
...
 <li> Azure Workload Backup only support TLS 1.2 . Change TLS registry settings as described <a href = 'https://docs.microsoft.com/windows-server/security/tls/tls-registry-settings'>here</a> to enable TLS 1.2 .</li>
```
Comment (kajai987) [inline ConnectivityTestScriptsForWindows/Publish-Report.ps1:80]: supports

Comment (kajai987) [reply ConnectivityTestScriptsForWindows/Publish-Report.ps1:80]: remove space before the full stop everywhere

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 [hunk 7]
```diff
@@ -65,8 +78,16 @@ $AzureBackupCommonIssue =
...
-<li> If not using Private Endpoint, check Network Security Group (NSG) attached to subnet, NSG should allow outbound connection to Service Tag AzureBackup <a href = 'https://docs.microsoft.com/azure/virtual-network/service-tags-overview'>Service Tags. </a>
-Check firewall settings for outbound connection to URLs mentioned in <a href='https://docs.microsoft.com/azure/backup/backup-sql-server-database-azure-vms#allow-access-to-service-fqdns'> this section </a></li>
+<li> If not using Private Endpoint, check Network Security Group (NSG) attached to subnet, NSG should allow outbound connection to AzureBackup <a href = 'https://docs.microsoft.com/azure/virtual-network/service-tags-overview'>Service Tags. </a>
```
Comment (kajai987) [inline ConnectivityTestScriptsForWindows/Publish-Report.ps1:81]: If the Azure Recovery Services vault is not using Private Endpoint....

Comment (kajai987) [reply ConnectivityTestScriptsForWindows/Publish-Report.ps1:81]: to the subnet

Comment (kajai987) [reply ConnectivityTestScriptsForWindows/Publish-Report.ps1:81]: check the Network Security...

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 [hunk 8]
```diff
@@ -65,8 +78,16 @@ $AzureBackupCommonIssue =
...
+</br>
+If the rule is missing, Click on 'Add outbound port rule' and fill in the source as Virtual network, destination as Service tag and Destination Service tag as AzureBackup. Set Action as Allow and click Add.
+Specific ports for this connection can be found in <a href='https://docs.microsoft.com/azure/backup/backup-sql-server-database-azure-vms#allow-access-to-service-fqdns'> this section </a>
+</br>
+<img src='media/AddOutboundRuleBackup.png' alt='Add outbound port rule blade for Azure backup' height=480 width=256/>
+</br>
+Check your firewall settings for outbound connection to URLs mentioned in <a href='https://docs.microsoft.com/azure/backup/backup-sql-server-database-azure-vms#allow-access-to-service-fqdns'> this section </a></li>
 <li> If the connection status for service contains 'The remote name could not be resolved', ensure the DNS is configured properly to resolve the required URLs. If you are using private endpoint, guidance for DNS management is documented <a href ='https://docs.microsoft.com/azure/backup/private-endpoints#manage-dns-records'>here</a></li>
```
Comment (kajai987) [inline ConnectivityTestScriptsForWindows/Publish-Report.ps1:91]: ensure that the

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 [hunk 9]
```diff
@@ -291,7 +320,7 @@ $html = "
     <br><br>
     <h3 style=""background-color:powderblue; font-family:helvetica;""> Proxy Settings </h3>
     <div style='background-color:#007FFF;color:white;padding:20px;'>
-    <h3> <b> If Proxy Settings are not common for all users, ensure the settings for user NT Authority\System and NT Service\AzureWLBackupPluginSvc are same. Run Set-ProxySettingsForPluginUser.ps1 with NT Authority\System user context to configure settings for NT Service\AzureWLBackupPluginSvc</b></h3>
+    <h3> <b> If Proxy Settings are not common for all users, ensure the settings for user NT Authority\System and NT Service\AzureWLBackupPluginSvc are same. Run Set-ProxySettingsForPluginUser.ps1 as NT Authority\System user to set the settings for NT Service\AzureWLBackupPluginSvc user same as LocalSystem user</b></h3>
```
Comment (kajai987) [inline ConnectivityTestScriptsForWindows/Publish-Report.ps1:323]: ensure that the

Comment (kajai987) [reply ConnectivityTestScriptsForWindows/Publish-Report.ps1:323]: Mention that they can use psexec tool for running commands under NT Authority/System. Mention the steps

**Review by kajai987 — CHANGES_REQUESTED** [review]
Given comments

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 (hunk 1 above)
Comment (shashwattrivedi) [reply ConnectivityTestScriptsForWindows/Publish-Report.ps1]: Changed

Comment (shashwattrivedi) [reply ConnectivityTestScriptsForWindows/Publish-Report.ps1]: Changed

Comment (shashwattrivedi) [reply ConnectivityTestScriptsForWindows/Publish-Report.ps1]: made it consistent to the one on the portal

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 (hunk 2 above)
Comment (shashwattrivedi) [reply ConnectivityTestScriptsForWindows/Publish-Report.ps1]: Removed

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 (hunk 5 above)
Comment (shashwattrivedi) [reply ConnectivityTestScriptsForWindows/Publish-Report.ps1]: Done

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 (hunk 7 above)
Comment (shashwattrivedi) [reply ConnectivityTestScriptsForWindows/Publish-Report.ps1]: Done

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 (hunk 9 above)
Comment (shashwattrivedi) [reply ConnectivityTestScriptsForWindows/Publish-Report.ps1]: Added the steps

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 (hunk 6 above)
Comment (shashwattrivedi) [reply ConnectivityTestScriptsForWindows/Publish-Report.ps1]: Removed

File: ConnectivityTestScriptsForWindows/Publish-Report.ps1 (hunk 8 above)
Comment (shashwattrivedi) [reply ConnectivityTestScriptsForWindows/Publish-Report.ps1]: changed

**Review by shashwattrivedi — COMMENTED** [review]
Resolved comments

**Pull request merged**